packages/fhir-converter/src/lib/handlebars-converter/handlebars-helpers.js

```diff
@@ -147,6 +147,30 @@ var alreadyValidDateTime = function (dateTimeString) {
   );
 };
 
+const incompeteIso8601Regex = /^\d{4}-\d{2}-\d{2} \d{2}:\d{2}:\d{2}$/;
+const Iso8601RegexWithMissingT = /^\d{4}-\d{2}-\d{2} \d{2}:\d{2}:\d{2}(\.\d{3}Z)$/;
+
+var incompleteIso8601 = function (dateTimeString) {
+  if (incompeteIso8601Regex.test(dateTimeString) || Iso8601RegexWithMissingT.test(dateTimeString)) {
+    return true;
+  }
+  return false;
+};
+
+var correctIsoFormat = function (dateTimeString) {
+  if (Iso8601RegexWithMissingT.test(dateTimeString)) {
+    const newDate = dateTimeString.replace(" ", "T");
```

Comment:
Prob good to add a `.trim()` before the replace, both here and below.

Also, should we always check for `undefined` on these functions? Can't really tell if we're getting something or not (JS sucks).

Reply:
both good points! 🙇